Issue when user path contains spaces

Extension path contains spaces because of MS Windows profile name - `C:\Users\Mateusz Zieliński\.vscode\extensions\nicolasvuillamy.vscode-groovy-lint-1.1.0`

```
Start analyzing file:///c%3A/git/jenkins-pipeline-library/vars/continuousIntegrationPipeline.groovy
Unable to run java command: {"status":1,"stdout":"","stderr":""}
GroovyLint: Error running CodeNarc: 
Error: Could not find or load main class Zieli�ski\.vscode\extensions\nicolasvuillamy.vscode-groovy-lint-1.1.0\server\node_modules\npm-groovy-lint\lib\java\CodeNarc-1.6.1.jar;c:\Users\Mateusz
Caused by: java.lang.ClassNotFoundException: Zieli�ski\/vscode\extensions\nicolasvuillamy/vscode-groovy-lint-1/1/0\server\node_modules\npm-groovy-lint\lib\java\CodeNarc-1/6/1/jar;c:\Users\Mateusz

===========================================================================
===========================================================================
npm-groovy-lint error: Fatal error while calling CodeNarc
Reason: unknown
Error: Could not find or load main class Zieli�ski\.vscode\extensions\nicolasvuillamy.vscode-groovy-lint-1.1.0\server\node_modules\npm-groovy-lint\lib\java\CodeNarc-1.6.1.jar;c:\Users\Mateusz
Caused by: java.lang.ClassNotFoundException: Zieli�ski\/vscode\extensions\nicolasvuillamy/vscode-groovy-lint-1/1/0\server\node_modules\npm-groovy-lint\lib\java\CodeNarc-1/6/1/jar;c:\Users\Mateusz

undefined
If you still have an error, post an issue to get help: https://github.com/nvuillam/vscode-groovy-lint/issues
===========================================================================
===========================================================================

```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -1,94 +1,94 @@
-  version: 2.1
+version: 2.1
 
-  orbs:
-    win: circleci/windows@2.2.0
+orbs:
+  win: circleci/windows@2.2.0
 
-  jobs:
-    debian_jdk8: 
-      docker:
-        - image: circleci/openjdk:8-jdk-buster-node-browsers
-      steps:
-        - checkout
-        - run:
-            name: Install dependencies
-            command: npm install --unsafe-perm
-        - run:
-            name: Lint
-            command: npm run lint
-        - run:
-            name: Compile
-            command: npm run compile
-        - run:
-            name: Test & Code Coverage
-            command: npm run test:coverage
-        - run:
-            name: Submitting code coverage to codecov
-            command: |
-              ./node_modules/.bin/nyc report --reporter text-lcov > coverage.lcov
-              curl -s https://codecov.io/bash | bash
+jobs:
+  debian_jdk8:
+    docker:
+      - image: circleci/openjdk:8-jdk-buster-node-browsers
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: npm install --unsafe-perm
+      - run:
+          name: Lint
+          command: npm run lint
+      - run:
+          name: Compile
+          command: npm run compile
+      - run:
+          name: Test & Code Coverage
+          command: npm run test:coverage
+      - run:
+          name: Submitting code coverage to codecov
+          command: |
+            ./node_modules/.bin/nyc report --reporter text-lcov > coverage.lcov
+            curl -s https://codecov.io/bash | bash
 
-    debian_jdk11: 
-      docker:
-        - image: circleci/openjdk:11-jdk-buster-node-browsers
-      steps:
-        - checkout
-        - run:
-            name: Install dependencies
-            command: npm install --unsafe-perm
-        - run:
-            name: Lint
-            command: npm run lint
-        - run:
-            name: Compile
-            command: npm run compile
-        - run:
-            name: Test 
-            command: npm run test
+  debian_jdk11:
+    docker:
+      - image: circleci/openjdk:11-jdk-buster-node-browsers
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: npm install --unsafe-perm
+      - run:
+          name: Lint
+          command: npm run lint
+      - run:
+          name: Compile
+          command: npm run compile
+      - run:
+          name: Test
+          command: npm run test
 
-    debian_jdk13: 
-      docker:
-        - image: circleci/openjdk:13-jdk-buster-node-browsers
-      steps:
-        - checkout
-        - run:
-            name: Install dependencies
-            command: npm install --unsafe-perm
-        - run:
-            name: Lint
-            command: npm run lint
-        - run:
-            name: Compile
-            command: npm run compile
-        - run:
-            name: Test 
-            command: npm run test:verbose
+  debian_jdk13:
+    docker:
+      - image: circleci/openjdk:13-jdk-buster-node-browsers
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: npm install --unsafe-perm
+      - run:
+          name: Lint
+          command: npm run lint
+      - run:
+          name: Compile
+          command: npm run compile
+      - run:
+          name: Test
+          command: npm run test:verbose
 
-    windows: 
-      executor:
-        name: win/default # executor type
-        size: "medium" # resource class, can be "medium", "large", "xlarge", "2xlarge", defaults to "medium" if not specified
-        shell: bash.exe
-      steps:
-        - checkout
-        - run:
-            name: Install dependencies
-            command: npm install --unsafe-perm
-        - run:
-            name: Lint
-            command: npm run lint
-        - run:
-            name: Compile
-            command: npm run compile
-        - run:
-            name: Test 
-            command: npm run test
- 
-  workflows:
-    version: 2
-    "vscode-groovy-lint-build-test":
-      jobs:
-        - debian_jdk8
-        - debian_jdk11
-        - debian_jdk13
-        - windows # Disable if CircleCI reached limit of free plan
+  windows:
+    executor:
+      name: win/default # executor type
+      size: "medium" # resource class, can be "medium", "large", "xlarge", "2xlarge", defaults to "medium" if not specified
+      shell: bash.exe
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: npm install --unsafe-perm
+      - run:
+          name: Lint
+          command: npm run lint
+      - run:
+          name: Compile
+          command: npm run compile
+      - run:
+          name: Test
+          command: npm run test
+
+workflows:
+  version: 2
+  "vscode-groovy-lint-build-test":
+    jobs:
+      - debian_jdk8
+      - debian_jdk11
+      - debian_jdk13
+      - windows # Disable if CircleCI reached limit of free plan
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.2.3] 2020-08-29
+
+- Fix bug on Windows when username contains space(s)
+
 ## [1.2.2] 2020-08-21
 
 - Allow user to hide future npm-groovy-lint error messages

--- a/README.md
+++ b/README.md
@@ -80,6 +80,10 @@ Please follow [Contribution instructions](https://github.com/nvuillam/vscode-gro
 
 ## Release Notes
 
+### [1.2.3] 2020-08-29
+
+- Fix bug on Windows when username contains space(s)
+
 ### [1.2.2] 2020-08-21
 
 - Allow user to hide future npm-groovy-lint error messages

--- a/server/package.json
+++ b/server/package.json
@@ -34,7 +34,7 @@
     "find-java-home": "^1.1.0",
     "fs-extra": "^8.1.0",
     "glob-promise": "^3.4.0",
-    "npm-groovy-lint": "^7.4.0",
+    "npm-groovy-lint": "^7.4.4",
     "vscode-languageserver": "^6.1.1",
     "vscode-uri": "^2.1.1"
   },

